opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/services/ServiceContextProvider.java

```diff
@@ -64,15 +74,41 @@
         if (null != authSession) {
             return authSession.getParent();
         } else {
-            ServiceSession session = new ServiceSession();
+            FacadeServiceableRequest facadeServiceable = request.getFacadeServiceable();
+            String sessionPassword = facadeServiceable.getSessionPassword();
+
+            if (facadeServiceable.getXRequestID() == null) {
```

Comment:
getXRequestID is for logging, facade should not throw on it, REST api could

Reply:
I mixed up xRequestID and ServiceSessionID